Resources/config/services.xml

```diff
@@ -43,19 +43,6 @@
             <tag name="sulu.context" context="admin"/>
         </service>
 
-        <!-- FIXME hack to enable doctrine object constructor -->
-        <!--
-          could be fixed in serializer by changing JMS\SerializerBundle\DependencyInjection\Compiler\DoctrinePass:49
-          from "->replaceArgument(0, new Reference($previousId[$service]))"
-          to   "->replaceArgument(1, new Reference($previousId[$service]))"
-        -->
-        <service id="jms_serializer.object_constructor" alias="sulu_redirect.object_constructor" public="false"/>
-        <service id="sulu_redirect.object_constructor" class="JMS\Serializer\Construction\DoctrineObjectConstructor"
-                 public="false">
-            <argument type="service" id="doctrine"/>
-            <argument type="service" id="jms_serializer.unserialize_object_constructor"/>
-        </service>
```

Comment:
deserialization is not longer used and this is not compatible with JMS serializer 5 so removing it